Grace.py

```diff
@@ -0,0 +1,40 @@
+#A task was given before to just include a name and then parse it of which I did 
+#The second assignment was to parse both firstname and lastname together which was
+#done and Favour Samuel was gotten.
+#The third task isto Capitalise the first letters of "hello amazing grace"
+
+#Code for the second task.
+import argparse

```

Comment:
You can remove this, since you import `ArgumentParser` below.

Reply:
I separated the folders now.